modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -439,6 +439,11 @@
 			"jsp"
 		]
 	},
+	{
+		"from": "ServiceTrackerList<Class<T>, Class<E>>",
+		"issueKey": "LPS-195079",
+		"to": "ServiceTrackerList<typeParam#0#>"
+	},
```

Comment:
@TamyrisTorres, this is not sort according to the `issueKey`, can you please fix?